### PR TITLE
Update release version and migrate code

### PR DESCRIPTION
**Pull request by amityadav0**
- Update release version
- Enable version checks while migration

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+export VERSION=$(git describe --tags --match 'v*' --abbrev=0 | sed 's/^v//')
+
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/account-history-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/trade-shield-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/financial-snapshot/Cargo.toml
```
Comment (cosmic-vagabond) [inline scripts/build.sh:7]: @amityadav0 is it possible to only call this in a CI environment (`$CI == true`) ?

File: scripts/fast_build.sh [hunk 2]
```diff
@@ -11,6 +11,12 @@ mkdir -p artifacts
 
 rm -rf ./target/wasm32-unknown-unknown/
 
+export VERSION=$(git describe --tags --match 'v*' --abbrev=0 | sed 's/^v//')
+
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/account-history-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/trade-shield-contract/Cargo.toml
+sed -i '' "s/^version = .*/version = \"$VERSION\"/" contracts/financial-snapshot/Cargo.toml
```
Comment (cosmic-vagabond) [inline scripts/fast_build.sh:18]: @amityadav0 same here

File: scripts/build.sh (hunk 1 above)
Comment (amityadav0) [reply scripts/build.sh]: yes

File: scripts/fast_build.sh (hunk 2 above)
Comment (amityadav0) [reply scripts/fast_build.sh]: done

**Review by cosmic-vagabond — APPROVED** [review]

**Pull request merged**